demos/LowAllocationWebServer/IFormatterHttpExtensions.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Text.Formatting;
+
+namespace System.Net.Http.Buffered
+{
+    public static class IFormatterHttpExtensions
+    {
+        private const string Http = "HTTP/";
```

Comment:
This should be optimized, i.e. when writing as UTF8 the API would use Utf8String literal. Please feel free to add a todo and a separate workitem to do it later.